vignettes/tidyverse_translation.Rmd

```diff
@@ -30,7 +30,7 @@ pkgs <- c(
 # a logical that is FALSE only if deps are not installed (cf easystats/easystats#317)
 evaluate_chunk <- TRUE
 
-if (!all(vapply(pkgs, requireNamespace, quietly = TRUE, FUN.VALUE = logical(1L)))) {
+if (!all(vapply(pkgs, requireNamespace, quietly = TRUE, FUN.VALUE = logical(1L))) || getRversion() < "4.2.0") {
```

Comment:
This should be < 4.1 since the base pipe appeared in 4.1.0.

Also, that would solve the issue in CRAN checks but it means that any user with 3.6 or 4.0 wouldn't be able to see the vignettes, which seems unfair (since we say we support those versions).

Reply:
I'm not sure if the users won't see it, since vignettes are compiled and available on the CRAN and GitHub websites?

Reply:
I don't know if users would be able to see it or not, but in any case someone on 3.6 or 4.0 wouldn't be able to only copy-paste the code, they would also need to replace the native pipe by `magrittr`'s one. I think we can wait for the moment our oldest support version is 4.1 before making this change.

Reply:
Can you add a note at the top of the vignette saying that users on 3.6 or 4.0 should replace the native pipe by `magrittr`'s one?